client/my-sites/plans/plan-overview/plan-upgrade/index.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import i18n from 'lib/mixins/i18n';
+import { cartItems } from 'lib/cart-values';
+import CompactCard from 'components/card/compact';
+import { isPremium } from 'lib/products-values';
+import * as upgradesActions from 'lib/upgrades/actions';
+
+const PlanUpgrade = ( { plan, selectedSite: { slug: selectedSiteSlug } } ) => {
+	const upgradePlan = ( event ) => {
+		event.preventDefault();
+
+		upgradesActions.addItem( cartItems.planItem( 'business-bundle', true ) );
+
+		page( `/checkout/${ selectedSiteSlug }` );
+	};
+
+	if ( isPremium( plan ) ) {
+		return (
+			<CompactCard className="plan-upgrade">
+				{ i18n.translate( 'Need more power for your site? {{a}}Switch to the WordPress.com Business Free Trial.{{/a}}', {
+					components: { a: <a href="#" onClick={ upgradePlan } /> }
+				} ) }
+			</CompactCard>
+		);
+	}
+
+	// functional stateless components must return a ReactElement
+	return <noscript />;
```

Comment:
This seems appropriate because React returns `<noscript />` for React.Component classes that return null from their render(): https://github.com/facebook/react/issues/5355#issuecomment-152949327